assets/scss/base/_icons.scss

```diff
@@ -114,6 +114,29 @@ Styleguide Icons.Exclamation
 }
 
 /*
+Tick
+
+The tick icon is used to highlight actioned content to the user.
+
+Markup:
+<i class="tick"></i>
```

Comment:
Do screen readers need to know about an image of a tick? 

Apart from `class`, I'd say `aria-hidden="true"` is the only attribute you need here.